tests/BookmarksManager.Tests/NetscapeReaderTests.cs

```diff
@@ -82,7 +84,7 @@ public void FolderAttributesContainer()
             var html = string.Format(HtmlTemplate, "<DT><H3 ADD_DATE=1355307132 custom='1'>Test folder</H3><DL><p></DL><p>", "utf-8");
             var container = _reader.Read(html);
             var folder = (BookmarkFolder) container.Single();
-            Assert.AreEqual(new DateTime(2012, 12, 12, 12, 12, 12, DateTimeKind.Utc), folder.Added, "Folder attributes was not readed");
+            Assert.AreEqual(new DateTime(2012, 12, 12, 11, 12, 12, DateTimeKind.Local), folder.Added, "Folder attributes was not readed");
```

Comment:
Test fails with following message otherwise:
> Expected:<12.12.2012. 12:12:12>. Actual:<12.12.2012. 11:12:12>

So in order to fix it I switch `Kind `to `DateTimeKind.Local` to align it with `Kind `of converted _ADD_DATE_ attribute's value. That value is converted using following method which converts `DateTime `in UTC to Local time.

```
public static DateTime? FromUnixTimeStamp(string unixTimeStamp)
        {
            int unixTime;
            if (!string.IsNullOrEmpty(unixTimeStamp) && int.TryParse(unixTimeStamp, out unixTime))
            {
                return new DateTime(1970, 1, 1, 0, 0, 0, 0, DateTimeKind.Utc).AddSeconds(unixTime).ToLocalTime();
            }
            return null;
        }
```
Since you said UTC is used throughout the project could you once again check `DateTimeHelper `and see whether we need to fix its methods.

Reply:
Sorry, I was mistaken, UTC was not used everywhere. Fixed now.

Reply:
Thx for the fix, just small note that I'm going to vacation tomorrow, so you can expect other things to be fixed/changed after new year :)

Reply:
@darjanbogdan 
I think we agreed on everything. So maybe I can make fixes myself and merge your PR? I'm back in the context anyway :)
Also, I'm planning to change DateTime to DateTimeOffset, just to avoid UTC confusion in the future.

Reply:
@Dissimilis Yes we did, so if you have time I don't mind if you apply those changes, otherwise I will do it. All in all, thank you very much for your effort! :)